compiler/dialects.py

```diff
@@ -308,11 +310,61 @@ def DecorateCombineRule(self, rule, var):
     return rule
 
 
+class Databricks(Dialect):
+    """Databricks dialect"""
```

Comment:
Can you please add a brief comment explaining current state of the dialect. I.e. what are critical features yet to be implemented?

E.g.:
1) Unnesting. Does this work?
```
Q(x) :- x in [1,2,3];
```


2) Records.
Does this work?
```
P({a:1, b: ["x","y"]});
```

I am assuming something is yet TODO, is that so?